virtualizor.py

```diff
@@ -165,7 +166,7 @@ def create_networks(self):
         net_definitions = {"%s_sps" % self._conf.prefix: {}}
         for netname in net_definitions:
             exists = netname in existing_networks
-            if exists and self._conf.replace:
+            if exists and self._conf.cleanup:
                 self.conn.networkLookupByName(netname).destroy()
                 logging.info("Cleaning network %s." % netname)
```

Comment:
nop the networks are not defined (we do not create the XML file on the disk). Maybe we should change that.